packages/database/src/core/Repo.ts

```diff
@@ -63,6 +63,7 @@ export class Repo {
   private nextWriteId_ = 1;
   private server_: ServerActions;
   private statsReporter_: StatsReporter;
+  // eslint-disable-next-line camelcase
```

Comment:
It looks like we can just rename it (but we have to rename the usage in Repo_transaction as well).